head/presence-sensing/tests/test_adc.cpp

```diff
@@ -0,0 +1,13 @@
+#include "catch2/catch.hpp"
+#include "presence-sensing/core/presence_sensing_driver.hpp"
+#include "presence-sensing/tests/mock_adc.hpp"
+
+SCENARIO("get readings called on ADC") {
```

Comment:
This test is just directly calling a mock. The only thing that it tests is that mock. It doesn't test anything about the actual code that will run on hardware. We should get rid of it.